### ISSUE TITLE
FTB Ranks support

### ISSUE THREAD

**Issue by jmadarg** [lucko/spark#360]
### Description

When managing permissions through FTB Ranks, there is no permission node available on the FTB Ranks Readme (when you do `/ftbranks refresh_readme` the `README.txt` for FTB Ranks updates, listing all the command permission nodes available to it).

Not really sure if this is something on the side of Spark or FTB Ranks, but having this feature would be really useful for server management.

This was tested on 1.19.2 and 1.20.1 Forge servers.

### Proposed Behaviour

Spark permission nodes to be shown in the FTB Ranks README, so they can be managed via that mod.

**Comment by 2chriswy** [comment]
Yes please do this, it would be very useful /ops are not always online and its good for players to see the info on /spark tps

**Comment by nichtDanger** [comment]
❤️ I would also be happy if it was implemented. I spent hours trying to get the permissions to work in vain.
